python/ray/train/callbacks/callback.py

```diff
@@ -1,13 +1,23 @@
 import abc
 from typing import List, Dict
 
-from ray.train.callbacks.results_preprocessors import ResultsPreprocessor
+from ray.train.callbacks.results_preprocessors import ResultsPreprocessor, \
+    ExcludedKeysResultsPreprocessor, SequentialResultsPreprocessor
+from ray.train.constants import PYTORCH_PROFILER_KEY
 
 
 class TrainingCallback(abc.ABC):
     """Abstract Train callback class."""
 
     results_preprocessor: ResultsPreprocessor = None
+    # Reserved keys used by across all Callbacks.
+    # By default these will be filtered out from ``train.report()``.
+    # See ``_preprocess_results`` for more details.
+    ALL_RESERVED_KEYS = {PYTORCH_PROFILER_KEY}
```

Comment:
Nit: I'm thinking this might make more sense as a constant at the module level or in the `constants.py` file rather than a class variable. There's no case where you'd need to do `self.ALL_RESERVED_KEYS`